src/components/navbar/NavBar.tsx

```diff
@@ -18,11 +18,11 @@ export default function NavBar(props: ISelfProps) {
     return (
         <div className={styles.headerContainer}>
             <div className={styles.headerWrapper}>
-                <IconLogo height={42} width={116} onClick={() => window.location.href = window.appContext.baseUrl}/>
+                <IconLogo onClick={() => window.location.href = window.appContext.baseUrl}/>
                 <div className={classes(styles.linksWrapper, !menuOpen && styles.linksHidden)}>
                     <div className={styles.menu} onClick={() => setMenuOpen(!menuOpen)}>
-                        {!menuOpen && <IconMenu height={42} width={42} color={'white'}/>}
-                        {menuOpen && <IconClose height={42} width={42} color={'white'}/>}
+                        {!menuOpen && <IconMenu/>}
+                        {menuOpen && <IconClose/>}
```

Comment:
ad. to the comment above. This approach can help in writing less code with less UI bugs.

Reply:
That`s correct